test/integration/helpers.js

```diff
@@ -130,7 +130,7 @@ function invokeSubMocha(args, fn, cwd) {
 
 function _spawnMochaWithListeners(args, fn, cwd) {
   var output = '';
-  var mocha = spawn(process.execPath, args, {cwd: cwd});
+  var mocha = spawn(process.execPath, args, {cwd: cwd, env: {}});
```

Comment:
this avoids a problem when using `DEBUG`

Reply:
But aren't there _other_ envvars that we _do_ need?

Reply:
> But aren't there other envvars that we do or might need?

It's *possible* we *might* need this in the future, but I can't think of any reason.  

If we can get away with ignoring `STDERR`, that's a better fix.  I'll try it.

- [ ] see if we can ignore `STDERR` entirely when spawning for integration tests